### ISSUE TITLE
Resolution on lockscreen V3.100

### ISSUE THREAD

**Issue by kaylashb** [WangDaYeeeeee/GeometricWeather#308]
Lockscreen layout has lines after the update, resolution distorts in the same highlighted area, it's fine after the screen unlocks..
![Screenshot_20220503-085233](https://user-images.githubusercontent.com/104820695/166415939-82456a5f-8aad-4825-9c69-2f9662acac26.jpg)

**Comment by papjul** [comment]
What's the link between Geometric Weather and your lockscreen? What did you set up exactly?

**Comment by kaylashb** [comment]
Hi
Prior to the upgrade Geometric weather was setup to used as the LIVE Wallpaper for both the lock screen and home screen, I've not changed any setting since the latest update was released.Since the upgrade to 3.100 it seems there are now lines that appeared in the lock screen, it seems to have something to do with the resolution.
    On Tuesday, May 3, 2022, 09:15:11 AM GMT+2, Julien Papasian ***@***.***> wrote:  
 
 


What's the link between Geometric Weather and your lockscreen? What did you set up exactly?

**Comment by kaylashb** [comment]
> What's the link between Geometric Weather and your lockscreen? What did you set up exactly?

To add if I downgrade to 3.013 the issue is not seen on the lock screen

**Comment by papjul** [comment]
Yes, I understand your issue now. I thought a blue rectangle had appeared on your lockscreen. It's rather minor but I will let @WangDaYeeeeee check the regression.

**Comment by kaylashb** [comment]
> Yes, I understand your issue now. I thought a blue rectangle had appeared on your lockscreen. It's rather minor but I will let @WangDaYeeeeee check the regression.

**Thanks**, the blue rectangle was to highlight the problematic area. Appreciate the feedback

**Comment by WangDaYeeeeee** [comment]
I saw the picture.

In 3.100, I added a smooth gradient background for every material weather animation. 
And it looks like, on some devices, if there is a semi-transparent dark image overlapping the gradient background, it will generate an unnatural gradient, as the picture shows. This is largely related to the ability of the device to display colors.

Well maybe, I will set the gradient background appearance disabled by default in the next version, or maybe I will find a way to fix this problem.
